bulldozer/config_v1.go

```diff
@@ -68,6 +68,8 @@ type UpdateConfig struct {
 	Trigger Signals `yaml:"trigger"`
 	Ignore  Signals `yaml:"ignore"`
 
+	IgnoreDraftPR bool `yaml:"ignore_draft_pr"`
```

Comment:
This may need to be a `*bool` depending on how we decided to handle the situation where this is the only set field. See my top-level comment for context.

Reply:
My Go is a bit rusty, could you walk me through why `*bool` would be needed here if it's the only value set? Also how do you think we should handle this case in `bulldozer/update.go`? I'm trying to think through the cleanest implementation path I can

Reply:
Would it be to differentiate between a `nil` value for the config option vs implicit `false`?

Reply:
Yeah, this only needs to be a pointer if there's a reason to differentiate between "not set" and "set to false". With a simple `bool`, there's no way to distinguish between the zero value `false` and an explicit `false` in the config.

The line in `update.go` I mentioned might become something like this:

```go
if !updateConfig.Ignore.Enabled() && !updateConfig.Trigger.Enabled() && udpateConfig.IgnoreDrafts == nil {
	return false, nil
}
```

Then the condition later on would have to check for `nil` first:

```go
if updateConfig.IgnoreDrafts != nil && *updateConfig.IgnoreDrafts && pullCtx.IsDraft(ctx) {
  // ...
}
```

Reply:
Cool, that's the exact path I started implementing. Getting the tests right in `bulldozer/update_test.go` is a bit tricky though, as making `ignoreDrafts    *bool` in the `testMatrix` is giving some trouble setting the pointer reference to `true` or `false` boolean cleanly.

Reply:
I could add a `isIgnoreDraftsNil bool` and keep `ignoreDrafts` as `bool` but that doesn't seem very clean

Reply:
Adding a helper function like this is a common way to deal with "literal pointers" in tests:

```go
func boolp(b bool) *bool {
  return &b
}

// in a test
{ true, true, false, boolp(true) }
```

You could also define values and then reference them:

```go
valTrue, valFalse := true, false

// in a test
{ true, true, false, &valTrue }
```

Reply:
I think I'll go with the former, as the latter all pointing to the same value increases the likelihood of test state being manipulated across tests

Reply:
Pushed my attempt at this change, thanks for your ~pointers~ advice